### PR TITLE
[DEST-937] Update Fullstory crossorigin flag and add `fs.log`

### PR DESCRIPTION
**Pull request by butanian**
**What does this PR do?**
This PR updates the FS snippet to its current version. 

**Are there breaking changes in this PR?**
No

**Any background context you want to provide?**
Changes included in this version:
1. Enable cross-origin access to errors from fs.js
2. Make FS.log not throw an exception if it is called before FS is ready.

**Is there parity with the server-side/android/iOS integration components (if applicable)?**
Not applicable

**Does this require a new integration setting? If so, please explain how the new setting works**
No

**Links to helpful docs and other external resources**

### CODE REVIEW

File: integrations/fullstory/lib/index.js [hunk 1]
```diff
@@ -40,8 +40,11 @@ FullStory.prototype.initialize = function() {
   (function(m,n,e,t,l,o,g,y){
     if (e in m) {if(m.console && m.console.log) { m.console.log('FullStory namespace conflict. Please set window["_fs_namespace"].');} return;}
     g=m[e]=function(a,b,s){g.q?g.q.push([a,b,s]):g._api(a,b,s);};g.q=[];
+    o=n.createElement(t);o.async=1;o.crossOrigin='anonymous';o.src='https://'+_fs_host+'/s/fs.js';
+    y=n.getElementsByTagName(t)[0];y.parentNode.insertBefore(o,y);
```
Comment (sanjaybv) [inline integrations/fullstory/lib/index.js]: These two lines needn't be added. This integration loads `fs.js` on line 20.

File: integrations/fullstory/lib/index.js [hunk 2]
```diff
@@ -73,12 +76,13 @@ FullStory.prototype.identify = function(identify) {
 
   var newTraits = foldl(
     function(results, value, key) {
+      var rs = results;
```
Comment (sanjaybv) [inline integrations/fullstory/lib/index.js]: Just curious, why do we need this?

File: integrations/fullstory/lib/index.js (hunk 1 above)
Comment (butanian) [reply integrations/fullstory/lib/index.js]: removed, and added comment

File: integrations/fullstory/lib/index.js (hunk 2 above)
Comment (butanian) [reply integrations/fullstory/lib/index.js]: it was to get around the eslint no-param-reassign rule.

**Comment by mericsson** [comment]
This looks good to me. I'll start the release process earliest tomorrow afternoon and provide updates here.

@butanian if there are any more changes you'd like to make before releasing let me know. Thanks!

File: integrations/fullstory/lib/index.js (hunk 1 above)
Comment (jameremo) [reply integrations/fullstory/lib/index.js]: Apologies for the late review @sanjaybv, @butanian, and @mericsson, but some of line 43 here is important, specifically the `crossOrigin` and `async` attributes. Can we move those up to the script tag declaration on line 20?

**Comment by jameremo** [comment]
Many thanks @mericsson for the prompt review, and again sorry for the lack of promptness on my part!

File: integrations/fullstory/lib/index.js (hunk 1 above)
Comment (mericsson) [reply integrations/fullstory/lib/index.js]: @jameremo The line of code you commented on is from an old commit and the code is out of date. Not sure if this has been addressed or not?

**Comment by mericsson** [comment]
I see a late comment from @jameremo. I'll hold off on merging until I see review approval from @jameremo. Let me know if I can help answer any questions.

**Comment by butanian** [comment]
Thank you @jameremo !

 @mericsson I think we can go ahead and get started with the release process!

**Comment by mericsson** [comment]
Will start deploy process on Monday. Release expected to complete 8/14.

**Comment by jameremo** [comment]
Thank you @mericsson!

**Comment by mericsson** [comment]
Looks like due to concurrent analytics.js release this will be delayed with full release expected to complete Aug 20 hope that is OK.

**Comment by mericsson** [comment]
We will update here when it is complete.

**Comment by mericsson** [comment]
The last step of this deployment was complete at 15:29 PST today. It should now be rolled out to all customers. Let us know if you see any issues.

**Pull request merged**